utilities/getArticleByTitle/index.ts

```diff
@@ -0,0 +1,29 @@
+import { APPLICATION_JSON, CONTENTFUL_API_ACCESS_TOKEN, CONTENTFUL_API_URL, POST } from '../constants'
+import { ArticleProps } from '../../components/Article'
+
+export default async function getArticleByTitle(title: string): Promise<ArticleProps> {
+  const GET_ARTICLE_BY_TITLE_GRAPHQL_QUERY = `
+    {
+      articleCollection(where: {title: "${title}"}) {
+        items {
+          title
+          content
+        }
+      }
+    }
+  `
+
+  const res = await fetch(CONTENTFUL_API_URL, {
+    method: POST,
+    headers: {
+      'Content-Type': APPLICATION_JSON,
```

Comment:
Yeah if we get correct types for `fetch` then you shouldn't need to worry about typos / using constants here

Reply:
But the constant for the URL is good, right?

Reply:
Yeah the URL constant is still good 😁